ci/test_cpp.sh

```diff
@@ -34,13 +34,10 @@ nvidia-smi
 
 set +e
 
-# TODO: Disabling stream identification for now.
-# Set up library for finding incorrect default stream usage.
-#pushd "cpp/tests/utilities/identify_stream_usage/"
-#mkdir build && cd build && cmake .. -GNinja && ninja && ninja test
-#STREAM_IDENTIFY_LIB="$(realpath build/libidentify_stream_usage.so)"
-#echo "STREAM_IDENTIFY_LIB=${STREAM_IDENTIFY_LIB}"
-#popd
+# Get library for finding incorrect default stream usage.
+STREAM_IDENTIFY_LIB=$(conda info | grep "active env location" | awk 'BEGIN { FS = " : " } ; {print $2}')/lib/libcudf_identify_stream_usage.so
```

Comment:
```suggestion
STREAM_IDENTIFY_LIB="${CONDA_PREFIX}/lib/libcudf_identify_stream_usage.so"
```

This does what you're looking for. 

We use it for gtests on line 47 in this file.

I also just confirmed locally in the `rapidsai/ci:latest` image.

Reply:
Ah yes that's what I was looking for. Thanks!